### PR TITLE
fix pmap has no attribute raw bug

### PR DESCRIPTION
**Pull request by chlgit**

### CODE REVIEW

File: task_processing/plugins/mesos/execution_framework.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import threading
 import time
 from queue import Queue
-from typing import Optional
+from typing import Optional  # noqa, flake8 issue
```
[the remark this answers is not in the document]
Comment (chlgit) [reply task_processing/plugins/mesos/execution_framework.py]: Yeah, it's actually used at line 78 and 92. flake8 thinks it's unused wrongly.

**Comment by Rob-Johnson** [comment]
I'd rather we figured out where we're creating events without the raw event than just create a default - this is going to hide a lot of bugs. afaik, we should *always* be attaching the raw event to the event

**Comment by keymone** [comment]
agree with @Rob-Johnson, i would also go even further and make `raw` a mandatory field, so we crash at the location where we try to create malformed event.

**Comment by keymone** [comment]
see #112

**Pull request merged**